src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3973,6 +3994,7 @@ export class StudyViewPageStore
     }
 
     readonly unfilteredClinicalDataCount = remoteData<ClinicalDataCountItem[]>({
+        await: () => [this.filtersRemote],
```

Comment:
it seems that these filters/initialFilters are preconditions for the store.  if that's true then we should load them prior to instantiating store.  this dovetails again (i believe) with the change that was made for the hesitate PR.  lest discuss

Reply:
Fixed in [21bf551](https://github.com/cBioPortal/cbioportal-frontend/pull/4552/commits/21bf551a7500c17581aee854cd6af7987fe338dc), see also [this comment](https://github.com/cBioPortal/cbioportal-frontend/pull/4552/files/2006e93283a4ab1f98f1986685dc7c1c4b1269cc#r1168362636)